randovania/games/am2r/json_data/Distribution Center.txt

```diff
@@ -1666,17 +1620,29 @@ Extra - map_name: rm_a5b16
   * Open Passage to Spiky Space Jump Trial Entrance/Dock to Spiky Space Jump Trial
   > Dock to Spiky Space Jump Trial Reward Room
       Any of the following:
+          # TODO: HJ/DMG boost and DIBJ
           All of the following:
               # With Gravity Suit
               Gravity Suit
               Any of the following:
                   Space Jump
-                  Damage Boost (Advanced) and Walljump (Advanced) and Normal Damage ≥ 120
+                  Damage Boost (Advanced) and Walljump (Advanced) and Normal Damage ≥ 150
           All of the following:
               # Suitless
               Damage Boost (Expert) and Walljump (Hypermode) and Normal Damage ≥ 700
   > Pickup (Super Missile Tank)
-      Gravity Suit and Speed Booster and Shinesparking Tricks (Beginner)
+      All of the following:
+          Gravity Suit and Speed Booster and Shinesparking Tricks (Beginner)
+          Any of the following:
+              Disabled Entrance Rando
+              All of the following:
+                  # Short charge with the plant alive
+                  Short Charge (Intermediate)
+                  Ice Beam or Movement (Beginner)
+              All of the following:
+                  # Regular charge without the plant
```

Comment:
I'd change the comment to `Regular charge by killing plant` to make it a little bit more clear why ice and missiles are needed.
(Also, do the plants really need 2 missiles to get killed? weird choice, considering everythign else frozen just dies in one hit...)